test/local/variable-utilisation-rates-calculator.test.ts

```diff
@@ -0,0 +1,60 @@
+import {ethers, waffle} from 'hardhat'
+import chai from 'chai'
+import {solidity} from "ethereum-waffle";
+
+import VariableUtilisationRatesCalculatorArtifact
+  from '../../artifacts/contracts/VariableUtilisationRatesCalculator.sol/VariableUtilisationRatesCalculator.json';
+import {VariableUtilisationRatesCalculator} from "../../typechain";
+import {SignerWithAddress} from "@nomiclabs/hardhat-ethers/signers";
+import {fromWei, getFixedGasSigners, toWei} from "../_helpers";
+
+chai.use(solidity);
+
+const {deployContract} = waffle;
+const {expect} = chai;
+
+//From left: Utilisation, expected Borrowing Rate, expected Deposit Rate
+const TEST_TABLE =  [
+  [0,	0.05,	0],
```

Comment:
Formatting
+ Let's add some cases to test the precision (if it could be more than two digits) 
Especially we could add more test cases for the steep slope (currently we only got one point in the middle between breaking point and end of the line).